qiskit/scheduler/methods/basic.py

```diff
@@ -140,31 +142,50 @@ def translate_gates_to_pulse_defs(circuit: QuantumCircuit,
     circ_pulse_defs = []
 
     cmd_def = schedule_config.cmd_def
-    measured_qubits = set()  # Collect qubits that would like to be measured
+    qubit_mem_slots = {}  # Map measured qubit index to classical bit index
 
     def get_measure_schedule() -> CircuitPulseDef:
```

Comment:
I think this should be made a standalone method but we can do this in a future PR. What do you think?

Reply:
Ok, please make an issue:)

Reply:
Tracking in #3572.